### PR TITLE
Add two pods talking to each other monkey

### PR DESCRIPTION
**Pull request by nebril**

### CODE REVIEW

**Comment by raybejjani** [comment]
@nebril I don't think a reviewer was autoselected. You might need to add them manually.

**Comment by nebril** [comment]
@raybejjani can't assign them, I don't have rights

**Comment by nebril** [comment]
@tgraf can you look at it?

File: monkeys/clique/deployments/clique.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: chaos-clique-first
```
Comment (tgraf) [inline monkeys/clique/deployments/clique.yaml]: This needs a cooler name. You are even allowed to make a pin in this case. Use this opportunity wisely.

File: monkeys/clique/run.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+# run simple http server for other pod to connect to
+RESPONSE="HTTP/1.1 200 OK\r\nConnection: keep-alive\r\n\r\n${2:-"OK"}\r\n"
+while { echo -e "$RESPONSE"; } | nc -l -s 0.0.0.0 -p 80 -q 1 > /dev/null; do
+	:
+done &
+
+#make sure other pod is also up
+sleep 30
```
Comment (tgraf) [inline monkeys/clique/run.sh]: I wonder if we can make this more reliable?

**Comment by tgraf** [comment]
Merged via #6

**Pull request closed without merging**